src/lib/types.rs

```diff
@@ -2451,3 +2452,17 @@ pub struct CommandSpec {
     /// The command args
     pub args: Option<Vec<String>>,
 }
+
+pub fn either_to_result<L, R>(either: Either<L, R>) -> Result<L, R> {
+    match either {
+        Either::Left(lhs) => Ok(lhs),
+        Either::Right(rhs) => Err(rhs),
+    }
+}
+
+pub fn result_to_either<V, E>(result: Result<V, E>) -> Either<V, E> {
+    match result {
+        Ok(lhs) => Either::Left(lhs),
+        Err(rhs) => Either::Right(rhs),
+    }
+}
```

Comment:
who is using this?

Reply:
Oh that's leftover from when I was using `Either`; let me clean that up